packages/manager/apps/dedicated/client/app/components/dedicated-cloud/taskPollProgress/taskPollProgress.constant.js

```diff
@@ -0,0 +1,31 @@
+export const DEFAULT_TIMER = 5000;
+
+export const STATUS_POLL_READY = [
+  'doing',
```

Comment:
Why not use TASK_STATE values here ? Point of `TASK_STATE` is to abstract , right ?